packages/core/src/components/highlight/highlight.css

```diff
@@ -0,0 +1,72 @@
+.hds-highlight {
+  --accent-line-color: var(--color-bus);
+  --text-color: var(--color-black-90);
+  --variant-width-s: 264px;
+  --variant-width-m: 536px;
+  --variant-width-l: 680px;
+  --width: var(--variant-width-m);
+  --margin: var(--spacing-m) 0;
+
+  border-left: 8px solid var(--accent-line-color);
+  color: var(--text-color);
+  display: flex;
+  flex-direction: column;  
+  justify-content: center;
+  margin: var(--margin);
+  max-width: 100%;
+  padding: 0;
+  padding-left: var(--spacing-s);
+  row-gap: var(--spacing-m);
+  width: var(--width);
+}
+
+.hds-highlight > blockquote {
```

Comment:
I'm not a fan of element selectors just because browser finds elements by reading this selector line from right to left (so first target every blockquote element, after that only those with .hds-highlight parent), but I'm willing to let this one slip because who uses blockquotes anymore? 😁 I mean it's not a common element and this makes styling a bit easier in react component. Just wanted to speak my mind, feel free to give your two cents on the matter.